### DISCUSSION TITLE
快捷键的无动作只能定义一个么

### DISCUSSION THREAD

**Discussion by u2shana** [the1812/Bilibili-Evolved#4109] [提问 / Q&A] [answered]
想把播放器自带的d和m快捷键都禁用了,但是好像无动作只能添加一个

**Comment by Geeyun-JY3** [comment, marked as answer]
https://github.com/the1812/Bilibili-Evolved/blob/master/registry/lib/components/utils/keymap/help.md
> - 多个按键用空格隔开
